tools/setup_git_secrets.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
```

Comment:
I think since we have a shared hooks directory, we don't actually need this setup script. If you want to put a comment either in the README or in one of the hooks saying the hooks are auto-generated by `git secrets --install` I think that's good.